src/services/userService.js

```diff
@@ -0,0 +1,25 @@
+const { userDao } = require('../models');
+
+const isOrderValid = async (userId) => {
+  try {
+    const order = await userDao.userGetTickets(userId);
+
+    if (order && order.length > 0) {
+      return true;
+    }
+    return false;
+  } catch (error) {
+    console.error('Error checking order validity:', error);
+    throw error;
+  }
+};
+
+const userTickets = async (body) => {
+  const { userId } = body;
+  return await userDao.userGetTickets(userId);
```

Comment:
```suggestion
  const { userId } = body;
  const ticketInfo = userDao.getTicketInfoByUserId(userId);
  if (ticketInfo.length === 0) {
    throwError(400, 'INVALID_ORDER');
  }
  return await userDao.userGetTickets(userId);
```

1. valid한 ticket정보가 있는지 확인하는 과정이 userticket 정보를 가지고 오는 로직과 중복됩니다. 어차피 티켓 정보를 가져와서 길이를 확인하여 확인하실거라면, 티켓 정보를 가져오자마자 바로 예외 처리를 하시는 것이 좋을 것 같습니다.
2. 'invalid order id'라는 에러 메세지가 적합하지 않습니다. order id를 주고 받지 않는데, 혼선을 초래합니다.